enigma972/user-bundle/0.3/config/packages/user.yaml

```diff
@@ -0,0 +1,10 @@
+user:
```

Comment:
Lets change the name of this file to `enigma972_user.yaml`

Reply:
It's ok.